frontend/src/components/payment/cards/CardInfo.tsx

```diff
@@ -3,11 +3,11 @@ import { useTheme } from "next-themes"
 import { FunctionComponent, MouseEventHandler } from "react"
 import ReactCountryFlag from "react-country-flag"
 import { IMAGE_BASE_URL } from "../../../env"
-import { PaymentCard } from "../../../types/Payment"
 import Image from "next/image"
+import { CardInfo as TCardInfo } from "src/codegen"
```

Comment:
I should probably rename this type in the backend so it doesn't conflict with this frontend component name.

I think `WalletCard` is more descriptive anyway.